test/system/075-exec.bats

```diff
@@ -166,4 +166,38 @@ load helpers
     run_podman rm -f -t0 $cid
 }
 
+
+@test "podman exec --tty" {
+    run_podman run -d --name test $IMAGE top
+    run_podman 1 exec test printenv echo $TERM
```

Comment:
The `printenv echo $TERM` confuses me.  Do you want to `echo` _or_ `printenv`?

Reply:
Not only that, the `echo $TERM` (without single quotes) will interpret `$TERM` in the context of the running shell (BATS), not the container. This should be `printenv TERM`, as it is below.

Reply:
Yes this was a leak, attempting to get this, should all be printenv TERM.